app/models/concerns/type_of_means_assessment.rb

```diff
@@ -77,6 +77,18 @@ def benefit_check_recipient
 
   private
 
+  # involvement_in_case is stored on partner_detail when a database applications and
+  # partner when a datastore application.
+  def partner_involvement_in_case
+    partner_detail&.involvement_in_case || partner&.involvement_in_case
+  end
```

Comment:
I'm getting errors with this for historical apps where partner and partner detail are nil